database/query/pull_requests.sql

```diff
@@ -1,14 +1,9 @@
--- name: CreatePullRequest :one
```

Comment:
This query was only ever used in a test, and it could easily be replaced by the upsert operation without breaking the test.